### ISSUE TITLE
[Feature request] Rotate camera input with screen rotation

### ISSUE THREAD

**Issue by apandada1** [eupnea-linux/eupnea-utils#10]
Thanks for maintaining this great project.

I am using Kubuntu 23.04 with the ChromeOS kernel in a HP x360 14a (BLOOGUARD) chromebook.
When I flip the keyboard, the chromebook goes into tablet mode, and the touchscreen rotates flawlessly if I rotate the Chromebook.
However, the webcam input does not rotate, resulting in upside down or rotated images when the screen is rotated.

It would be great if Eupnea includes a screen to rotate the camera when the screen is rotated.

I tried the same in ChromeOS, and the camera does rotate when the screen is rotated.

(I am unsure if this is the correct place to ask for this feature request).

**Comment by Apacelus** [comment]
For clarification, in ChromeOS, the camera automatically rotates without any user input?

**Comment by apandada1** [comment]
Yes, if the screen is rotated, the camera rotates automatically in ChromeOS, without any user input. 

I wrote a script to rotate the camera in Linux using https://github.com/umlaeute/v4l2loopback (I will post my script soon). It would be great Eupnea/Depthboot includes a script to automatically rotate the camera.

**Comment by apandada1** [comment]
Here is how to rotate the webcam.

First install [`v4l2loopback-dkms`](https://archlinux.org/packages/?name=v4l2loopback-dkms) and the kernel headers (`linux-headers` in Arch).

Then, run `sudo modprobe v4l2loopback exclusive_caps=1` to enable it (you have to do it after every restart unless you [automatically load it with at boot](https://github.com/umlaeute/v4l2loopback#load-the-module-at-boot)).

Afterwards, run the command `v4l2-ctl --list-devices` to get the address of your actual webcam (`/dev/video0
` for me) and dummy webcam (`/dev/video1`).

Run the following command to project a rotated version of the actual webcam (`/dev/video0`) into the dummy webcam (`/dev/video1`). Keep the terminal open.
```
ffmpeg -f v4l2 -i /dev/video1 -vf "transpose=2,transpose=2,format=yuv420p" -f v4l2 
```

Finally, choose `Dummy video device` in any app that requires webcam, and you would see a rotated webcam image.
